modules/python/python-value-pairs.c

```diff
@@ -51,7 +51,7 @@ add_string_to_dict(PyObject *dict, const gchar *name, const char *value, gsize v
 {
   gchar buf[256];
 
-  PyObject *pyobject_to_add = PyBytes_FromStringAndSize(value, value_len);
+  PyObject *pyobject_to_add = _py_string_from_string(value, value_len);
```

Comment:
We discussed with Attila that there are some places that we want to do this conversion too. For example in `_py_log_message_subscript`.

Reply:
Done :)